### PR TITLE
[Work in Progress] adding SurfaceLR 

### PR DESCRIPTION
**Pull request by danyaljj**

### CODE REVIEW

File: surfaceLR-baseline/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ Follow the instructions below: 
+ 
+ - Unzip the folder which contains the liblinear files `liblinear.zip`. 
+ - Download the training dataset forom the [website](http://cogcomp.org/multirc/) and train the system with the training data. 
+ - To evaluate the system on the test data, run the command below: 
+ 
+ 
+ 
+ 
+
```
Comment (danyaljj) [inline surfaceLR-baseline/README.md]: @microth can you help me complete the instructions here?

Comment (danyaljj) [reply surfaceLR-baseline/README.md]: If it's easier, push your changes directly to this branch.

Comment (microth) [reply surfaceLR-baseline/README.md]: I revised the scripts a little, extended the README, and tried to push my changes. But I only get a "Permission to CogComp/multirc.git denied to microth"...

Comment (microth) [reply surfaceLR-baseline/README.md]: @danyaljj Can you check whether I have the rights necessary to push my changes?

Comment (danyaljj) [reply surfaceLR-baseline/README.md]: Should be fixed now.

Comment (microth) [reply surfaceLR-baseline/README.md]: Nope, still the same issue.

Comment (danyaljj) [reply surfaceLR-baseline/README.md]: Oops wrong person (https://github.com/miroth). Try 2, should be fixed now.

Comment (microth) [reply surfaceLR-baseline/README.md]: Yep, that worked. Changes committed.

**Comment by danyaljj** [comment]
Looks good. Merging.

**Pull request merged**